org.eclipse.winery.repository.rest/src/main/webapp/WEB-INF/web.xml

```diff
@@ -49,6 +49,7 @@
                 io.swagger.jaxrs.json,
                 io.swagger.jaxrs.listing,
                 org.eclipse.winery.repository.rest.resources
+                org.eclipse.winery.repository.rest.filters
```

Comment:
Shouldn't there be a comma before the new entry?

Reply:
I have corrected it